### DISCUSSION TITLE
Porting VBI Skip to Dolphin mmjr1 v11505

### DISCUSSION THREAD

**Discussion by ivan89el** [Medard22/Dolphin-MMJR2-VBI#56] [Ideas]
Hello! Maybe you could port VBI skip to this version? It's just that the assembly is the fastest of all that I tested and this hack could speed it up even more, it would be useful for "potato" devices on old or budget processors.

**Comment by Medard22** [comment]
That is not something that can be easily done. At least not by me. VBI Skip is actually just the tip of the iceberg. It relies on a changes and refactors to Dolphin timing. Those on the other side rely on other changes and refactors and the more you go back in time (back to MMJR1 codebase), the more it piles up.  I started this project because the difference between latest MMJR2 codebase and time when VBI Skip was added was couple of moths at most. And thanks to the Lumince´s updated source code it was eventually even less. 

Sorry I really cannot do what you ask, MMJR1 is just too old. It would need a skilled developer with a good knowledge of C++ and Dolphin internals and even then it probably wouldn´t be a walk in the park.

**Reply by ivan89el** [reply]
Well

**Reply by CaryllSimpas** [reply]
You try to ask gamer64 hoping he could help you port that on mmjrv1. Gamer64, himself updated the mmjr v1 to 11505 version. The last version of mmjr1
